src/mono/mono/mini/jit-icalls.c

```diff
@@ -1348,6 +1348,29 @@ mono_get_native_calli_wrapper (MonoImage *image, MonoMethodSignature *sig, gpoin
 	return compiled_ptr;
 }
 
+gpointer
+mono_gsharedvt_constrained_call_fast (gpointer mp, MonoGsharedvtConstrainedCallInfo *info, gpointer *out_receiver)
+{
+	switch (info->call_type) {
+	case MONO_GSHAREDVT_CONSTRAINT_CALL_TYPE_VTYPE:
+		/* Calling a vtype method with a vtype receiver */
+		*out_receiver = mp;
+		return info->code;
+	case MONO_GSHAREDVT_CONSTRAINT_CALL_TYPE_REF:
+		/* Calling a ref method with a ref receiver */
+		out_receiver = *(gpointer*)mp;
```

Comment:
Shouldn't the `out_receiver` be dereferenced here i.e.:
``` c
*out_receiver = *(gpointer*)mp;
```

Reply:
Yes, thanks